### PR TITLE
allow configurable openshift-web-console image

### PR DESCRIPTION
**Pull request by philbrookes**
Deploy custom web-console image.

## Verification steps:
1) Install this version of the addon into minishift and enable it.
2) reconfigure minishift addon-env
```
minishift config set addon-env CONTAINER_REPO_USERNAME=${DOCKERHUB_USER},CONTAINER_REPO_PASSWORD="${DOCKERHUB_PASSWORD}",CONSOLE_IMAGE=aerogear/origin-web-console:integrations
```
3) recreate minishift
```
minishift stop
minishift delete
minishift start --openshift-version v3.9.0 --service-catalog
```
4)
Deploy fh-sync-server to a project and you should see the integrations section in the overview once fh-sync-server has successfully deployed.

### CODE REVIEW

File: mobile-core.addon
Comment (david-martin) [inline mobile-core.addon]: we could make the query more resilient by using labels to find the pod.

e.g. 
```
oc get pods -n openshift-web-console -l app=openshift-web-console --template "{{(index .items 0).status.phase}}"  | grep Running

Running
```

File: docs/create-custom-console-container.adoc
Comment (david-martin) [inline docs/create-custom-console-container.adoc:18]: This cmd gives an error about a missing dep

```
CONSOLE_REPO_PATH=~/work/origin-web-console make vendor-console
GIT_REF= CONSOLE_REPO_PATH=/home/dmartin/work/origin-web-console hack/vendor-console.sh
Making sure go-bindata binary has been built...
go-bindata/main.go:15:2: cannot find package "github.com/jteeuwen/go-bindata" in any of:
	/usr/lib/golang/src/github.com/jteeuwen/go-bindata (from $GOROOT)
	/home/dmartin/go/src/github.com/jteeuwen/go-bindata (from $GOPATH)
[ERROR] PID 11454: hack/vendor-console.sh:23: `go install ./...` exited with status 1.
[INFO] 		Stack Trace: 
[INFO] 		  1: hack/vendor-console.sh:23: `go install ./...`
[INFO]   Exiting with code 1.
make: *** [Makefile:137: vendor-console] Error 1
```

**Review by david-martin — APPROVED** [review]
Approved. Tried running a custom image locally, and building a custom image. Both are good.

Good to merge if we get a 2nd ack from@sedroche

File: docs/create-custom-console-container.adoc
Comment (maleck13) [inline docs/create-custom-console-container.adoc:25]: Is there a link to this doc somewhere, I don't see it?

**Review by maleck13 — CHANGES_REQUESTED** [review]
I think we should very clearly call out in the README:

Setting up the mobile addon to view the latest UI changes with a set of instructions under that heading

**Pull request merged**
